apps/cli/src/types/command/environment.types.d.ts

```diff
@@ -0,0 +1,3 @@
+export interface EnvironmentData {
+  name: string
```

Comment:
There are other fields to the `Environment` body aswell. Do refer the controller in the API to understand their types.